.github/workflows/docker-build.yaml

```diff
@@ -149,13 +143,6 @@ jobs:
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@master
         if: success()
-        with:
-          image-ref: ${{ env.REGISTRY }}/${{ github.repository_owner }}/${{ matrix.Imagename }}:${{ github.sha }}
-          ignore-unfixed: true
-          exit-code: '1'
-          format: 'sarif'
-          output: 'trivy-results.sarif'
-          severity: 'CRITICAL,HIGH'
```

Comment:
I meant in my previous PR review to remove only `exit-code: '1'` the rest could stay

Reply:
Sorry if I tag wrongly the line

Reply:
Ok, what about: https://github.com/cloudoperators/greenhouse/blob/9710bcbc12fa3647322753d3b8ccdedd7d6a1060/.github/workflows/docker-build.yaml#L171 ?